### ISSUE TITLE
Pandoc required? 

### ISSUE THREAD

**Issue by InKyungChoi** [thoughthollow/UNECE_abst2word#1]
Is "pandoc" required? I got an error saying 
`Error: pandoc version 1.12.3 or higher is required and was not found (see the help page ?rmarkdown::pandoc_available).`

Checking before I install it...

**Comment by thoughthollow** [comment]
Yes, but if you're using RStudio then it comes bundled with it (Pandoc is a markup format converter that works with a variety of languages like markdown or LaTex to render documents, hence "pan").

Most likely something else has gone wrong with the rendering process (also referred to as knitting).

As a start, can you run sessionInfo() for me and report back?
Also, try to just run abst2word_V2.Rmd itself and tell me what happens.

**Comment by InKyungChoi** [comment]
> sessionInfo()
R version 4.4.3 (2025-02-28 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8    LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                           LC_TIME=English_United States.utf8    

time zone: Europe/Paris
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] officedown_0.4.0 janitor_2.2.1    readxl_1.4.5     lubridate_1.9.4  forcats_1.0.0    stringr_1.5.1    dplyr_1.1.4     
 [8] purrr_1.0.4      readr_2.1.5      tidyr_1.3.1      tibble_3.2.1     ggplot2_3.5.1    tidyverse_2.0.0 

loaded via a namespace (and not attached):
 [1] utf8_1.2.4              generics_0.1.3          fontLiberation_0.1.0    xml2_1.3.8              stringi_1.8.7          
 [6] digest_0.6.37           hms_1.1.3               magrittr_2.0.3          evaluate_1.0.3          grid_4.4.3             
[11] timechange_0.3.0        fastmap_1.2.0           cellranger_1.1.0        zip_2.3.2               scales_1.3.0           
[16] fontBitstreamVera_0.1.1 textshaping_1.0.0       cli_3.6.4               fontquiver_0.2.1        rlang_1.1.5            
[21] munsell_0.5.1           yaml_2.3.10             cachem_1.1.0            withr_3.0.2             gdtools_0.4.2          
[26] tools_4.4.3             officer_0.6.8           tzdb_0.5.0              uuid_1.2-1              memoise_2.0.1          
[31] colorspace_2.1-1        vctrs_0.6.5             R6_2.6.1                lifecycle_1.0.4         snakecase_0.11.1       
[36] ragg_1.3.3              pkgconfig_2.0.3         pillar_1.10.2           gtable_0.3.6            Rcpp_1.0.14            
[41] glue_1.8.0              systemfonts_1.2.2       rvg_0.3.5               xfun_0.52               tidyselect_1.2.1       
[46] knitr_1.50              htmltools_0.5.8.1       rmarkdown_2.29          compiler_4.4.3          askpass_1.2.1          
[51] openssl_2.3.2
